packages/components/src/Chip/components/ChipSuffix/Chip.Suffix.test.tsx

```diff
@@ -31,4 +32,77 @@ describe("Chip Compound Components", () => {
     const elem = container.querySelector("." + styles.empty);
     expect(elem).toBeInTheDocument();
   });
+
+  describe("when onClick prop is passed", () => {
+    it("should call onClick when suffix is clicked", () => {
+      const onClick = jest.fn();
+      const { getByTestId } = render(
+        <Chip.Suffix onClick={onClick} testID="ATL-Chip-Suffix">
+          <Icon name="cross" />
+        </Chip.Suffix>,
+      );
+
+      getByTestId("ATL-Chip-Suffix")?.click();
+      expect(onClick).toHaveBeenCalledTimes(1);
+    });
+
+    it("should call onClick when suffix is keypressed", async () => {
+      const onClick = jest.fn();
+      const { getByTestId } = render(
+        <Chip.Suffix onClick={onClick} testID="ATL-Chip-Suffix">
+          <Icon name="cross" />
+        </Chip.Suffix>,
+      );
+
+      const spanElement = getByTestId("ATL-Chip-Suffix");
+      expect(spanElement).not.toBeNull();
```

Comment:
While this is a valid assertion, I believe a more "semantic" one would be
```suggestion
      expect(spanElement).toBeInTheDocument();
```